examples/s3/bucket.yaml

```diff
@@ -46,5 +46,44 @@ spec:
             prefix: "ola/"
           expiration:
             days: 15
+    replicationConfiguration:
+      roleRef:
+        name: somerole
+      rules:
+        - destination:
+            storageClass: STANDARD
+            bucketRef:
+              name: repl-dest
+          deleteMarkerReplication:
+            status: Disabled
+          filter:
+            prefix: ""
```

Comment:
@chlunde This works for me both with and without this filter.

Reply:
cool, do you want me to remove it or keep it as it is?